install-config.yaml

```diff
@@ -23,7 +23,7 @@ networking:
     hostPrefix: 23
   machineNetwork:
   - cidr: 192.168.126.0/24
-  networkType: OpenShiftSDN
+  networkType: OVNKubernetes
```

Comment:
Maybe we should not specify it, and just use what OpenShift wants to use as a default?

Reply:
@cfergeau we have to check if we remove this field does it auto added or result to an error.

Reply:
See https://github.com/openshift/installer/blob/release-4.10/docs/user/customization.md :
```
networkType (optional string): The type of network to install. 
```